tree/master/cloud/src/deployments/Static Content/Matter Center Assets/Matter Landing Assets/scripts/MatterLanding.js

```diff
@@ -462,13 +463,37 @@ function displayContent() {
 			
 			sMatterInformationText  = sMatterInformationText.replace("@@Label1Tab1HeaderText", uiconfigs.MatterLanding.Label1Tab1HeaderText); 
 			sMatterInformationText  = sMatterInformationText.replace("@@Label2Tab2HeaderText", uiconfigs.MatterLanding.Label2Tab2HeaderText);
+			if(matterExtraProperties != null && matterExtraProperties != undefined && matterExtraProperties != '' )
+			{
+				sMatterInformationText  = sMatterInformationText.replace("@@ishide", "");
+				sMatterInformationText  = sMatterInformationText.replace("@@Label3Tab3HeaderText", uiconfigs.MatterLanding.Label3Tab3HeaderText);
+			}
+			else
+			{
+				sMatterInformationText  = sMatterInformationText.replace("@@ishide", "hide");
+				sMatterInformationText  = sMatterInformationText.replace("@@Label3Tab3HeaderText", "");
+			}
+
 			sMatterInformationText  = sMatterInformationText.replace("@@MyMatters", uiconfigs.MatterLanding.Label18MenuText);
 	        sMatterInformationText = sMatterInformationText.replace("title='My Matters'", "title='" + uiconfigs.MatterLanding.Label18MenuText + "'");
 			
 	
 				    	    	    	    	    	    
 		    $("#matterInfo").html(sMatterInformationText);
 		    $("#matterInfo").parent().append(oMatterLandingHtmlConstants.hierarchyHtml.replace(/@@MatterName/g, oMatterLandingCommonObjects.hierarchyLibraryName));
+		    
```

Comment:
Functionality to display Matter Properties by spliting MatterExtraProperties values injected at time of matter Landing Page creation.